tests/integration/test_commands.py

```diff
@@ -1,20 +1,1289 @@
+        assert sheet2["A1"].value == "Filename"
+        assert sheet2["B1"].value == "Sample ID"
+        assert sheet2["C1"].value == "File Format"
+        assert sheet2["D1"].value == "Component"
+        assert sheet2["E1"].value == "Genome Build"
+        assert sheet2["F1"].value == "Genome FASTA"
+
+        assert sheet2["A2"].value is None
+        assert sheet2["B2"].value is None
+        assert sheet2["C2"].value == "BAM"
+        assert sheet2["D2"].value is None
+        assert sheet2["E2"].value == "GRCh37"
+        assert sheet2["F2"].value is None
+
+        assert sheet2["A3"].value is None
+        assert sheet2["B3"].value is None
+        assert sheet2["C3"].value == "CRAM"
+        assert sheet2["D3"].value is None
+        assert sheet2["E3"].value == "GRCh38"
+        assert sheet2["F3"].value is None
+
+        assert sheet2["A4"].value is None
+        assert sheet2["B4"].value is None
+        assert sheet2["C4"].value == "CSV/TSV"
+        assert sheet2["D4"].value is None
+        assert sheet2["E4"].value == "GRCm38"
+        assert sheet2["F4"].value is None
+
+        assert sheet2["A5"].value is None
+        assert sheet2["B5"].value is None
+        assert sheet2["C5"].value == "FASTQ"
+        assert sheet2["D5"].value is None
+        assert sheet2["E5"].value == "GRCm39"
+        assert sheet2["F5"].value is None
+
+        # AND there are no more columns in the second sheet
+        assert sheet2["G1"].value is None
+
+        # A copy of the Excel file is saved to the test directory for manual verification
+        if testing_config.manual_test_verification_enabled:
+            path = os.path.join(
+                testing_config.manual_test_verification_path,
+                "CLI_TestManifestCommand_google_sheet_bulk_rna_with_annotations_url.txt",
+            )
+            with open(path, "w") as f:
+                f.write(google_sheet_url)
+
+    def test_generate_mock_component_excel_manifest(self, runner: CliRunner) -> None:
+        """
+        Tests for:
+        - Command has no errors, has exit code 0
+        - Command output has excel file message
+        """
+        try:
+            result = runner.invoke(
+                manifest,
+                [
+                    "--config",
+                    "tests/data/test_configs/CLI_test_config2.yml",
+                    "get",
+                    "--output_xlsx",
+                    "test-example.xlsx",
+                    "--dataset_id",
+                    "syn52746566",
+                ],
+            )
+            # Assert these files were created:
+            assert os.path.isfile("tests/data/example.MockComponent.schema.json")
+            assert os.path.isfile("test-example.xlsx")
+            workbook = load_workbook("test-example.xlsx")
+        finally:
+            # Remove created files:
+            os.remove("tests/data/example.MockComponent.schema.json")
+            os.remove("test-example.xlsx")
```

Comment:
I see in the TestDownloadManifest (Below) it was using this logic:

```
        try:
            os.remove(expected_manifest_file)
        except Exception:
            pass
```

If we run into issues we might need to do the same here. Or put the logic into a reusable function so we don't have a million try/except blocks.